docs/git/terminal/lesson.md

```diff
@@ -0,0 +1,216 @@
+## 1) Opening the Terminal
+
+There are a few different ways that you can open the terminal.
+
+### (Recommended) In Visual Studio Code
+
+Open VSCode and Click on `View` > `Terminal`.
+
+![The View Terminal Button](assets/viewterminal.png)
+
+This will then open a new Terminal windows
+
+![An example of a terminal window](assets/terminal.png)
+
+This is a fully featured Terminal where you can complete this week's coursework.
+
+### Through Your Operating System
+
+#### On Windows
+
+Open the Start Menu and search for `WSL`.
+
+Click on the `WSL` button to open the WSL window.
+
+This will open Windows Subsystem for Linux, it may take a moment to finish loading.
+
+#### On Mac
+
+Perhaps the easiest and quickest way to open Terminal is through Spotlight Search. To launch Spotlight, click the small magnifying glass icon in your menu bar (or press `Command+Space`).
+
+When the Spotlight Search bar pops up on your screen, type `terminal.app` and hit `Return`. Or you can click the Terminal.app icon that appears.
+
+Terminal will launch, and you’ll be ready to go.
+
+#### Ubuntu
+
+You’ll find the Terminal application included with your other installed applications. To find them, click the `Show Applications` button at the bottom-left corner of your screen, on the dash bar.
+
+Type `Terminal` and press Enter to find and launch the Terminal shortcut. You can also locate the Terminal icon in the list of all applications that appears here and click it.
+
+### Exercise (5 minutes)
+
+:::tip Exercise
+Using the above instructions, open the terminal now.
+
+Having issues? Can't open the terminal? Ask for help on your Slack channel **now**!.
+:::
+
+## 2) What is a GUI?
+
+GUI stands for **G**raphical **U**ser **I**nterface.
+
+A GUI is a useful way to make computers easier to use for general users of software. An example of a GUI is the Windows Desktop.
+
+![Windows 7 Desktop - Annotated](assets/windows-desktop.jpg)
+
+In a GUI, there a visual elements (buttons, text, windows etc) that a user can interact with to complete tasks.
+
+### Self-Study Exercise (15 minutes)
+
+:::tip Exercise
+Research the following questions
+
+- What is a GUI useful?
+- What are three examples of a GUI?
+- When were GUIs invented?
+- What came before GUIs?
+
+Anything you don't understand? Ask your questions in your Slack channel.
+:::
+
+## 3) What is a CLI (or Terminal)?
+
+CLI stands for **C**ommand **L**ine **I**nterface. An example of a CLI is Git Bash.
+
+![Example of a Terminal](assets/terminal-example.png)
+
+The terminal exists to take commands that you write and turn them into something that the computer can **do**.
+
+Many of the tasks that you complete using a GUI can also be done using the command line. For example,
+
+- Creating a file
+- Creating a folder
+- Looking at the contents of a file
+- Connecting to the internet
+- Opening programs
+
+You can write `commands` to your computer by wring text after the `$` sign in your terminal window and pressing `enter`.
```

Comment:
More recent versions of macOS default to `zsh` which uses a `%` symbol instead of a `$` symbol... I'm not sure how to best convey that without being confusing... Maybe just something like:
```suggestion
You can write `commands` to your computer by wring text after the `$` sign in your terminal window and pressing `enter`. (Some terminals use the `%` or `#` symbol instead of `$` - they work exactly the same).
```